spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -70,9 +73,11 @@ class Advert < ActiveRecord::Base
       end
     end
 
-    up, down = migrate
-    expect(up).to eq("remove_column :adverts, :published_at")
-    expect(down).to eq("add_column :adverts, :published_at, :datetime")
+    expect(migrate).to(
+      migrate_up("remove_column :adverts, :published_at").and(
+        migrate_down("add_column :adverts, :published_at, :datetime")
+      )
+    )
```

Comment:
It looks like we follow this `expect(x).to(migrate_up(y)).and(migrate_down(z))` pattern pretty often, and the syntax for it looks pretty easy to mess up imo. What do you think about adding a helper to take care of the syntax for you? 

```
def expect_migration_to_eq(migration, up:, down:)
   expect(migration).to(migrate_up(up)).and(migrate_down(down))
end
```

I'm torn because I like the way the current setup reads, so just wondering what your thoughts are on making it easier to write.

Reply:
I played around with that, but it ends up making the contract that much messier because you cant use inline `HEREDOCS` as nicely anymore. I think this pass is good enough and makes progress and if new patterns arise, we can continue to iterate.